java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/service/SourceCatalogueServer.java

```diff
@@ -76,8 +76,8 @@ public int execute(Namespace options, CommandLineApp app) {
 
         @Override
         public void addParser(ArgumentParser parser) {
-            parser.description("Create all topics that are missing on the Kafka server.");
-            parser.addArgument("-p" ,"-port")
+            parser.description("Port number of the Catalogue server.");
```

Comment:
This is the description of the task (I.e. web service).

Reply:
Indeed. My bad. Will correct